index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <!-- Section for menu toolbar -->
+    <section class="toolbar">
+        <div class="logo">
+            <h2>My logo</h2>
+        </div>
+        <div class="menu">
+            <img src="images/Union.png" alt="hamburger menu">
+        </div>
+    </section>
+    <!-- Main Body  -->
+    <section class="headline-section">
+        <h1 class="welcome">I'm Darren Glad to see you!</h1>
+
+        <p class="introduction">I’m a software developer! I can help you build a product 
+            , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <h3>LET'S CONNECT</h3>
+
+        <div class="flex-social">
+            <div><img src="vector/twitter.png" alt="Twitter"></div>
+            <div><img src="vector/linkedin_icon.png" alt="Linkedin"></div>
+            <div><img src="vector/medium.png" alt="Medium"></div>
+            <div><img src="vector/github.png" alt="Github"></div>
```

Comment:
- [ ] Please use a `ul` tag to group the social icons, and an `li` tag per icon, also wrap the `img` tag with an `a` tag because each icon is a link 💡